configuration/package_manifests/nfpm.tedge.yaml

```diff
@@ -133,3 +133,7 @@ contents:
     dst: /usr/share/tedge/diag-plugins/
     file_info:
       mode: 0755
+
+  - src: /usr/bin/tedge-file-log-plugin
+    dst: /usr/lib/tedge/log-plugins/file
+    type: symlink
```

Comment:
We've introduced an inconsistency here in the selection of the plugins path outside of the /etc/tedge folder.

* Change default plugin location from `/usr/lib/tedge/log-plugins` to `/usr/share/tedge/log-plugins` (to align with diag plugins)
* Allow the path to be configurable, for example the diag plugins can be configured for multiple directories, `diag.plugin_paths`.

The above is required to allow the plugin location to be configurable which is important when installing tedge on a read-only file systems, where tedge needs to be installed in a customer path (e.g. `/data/tedge`). Most notably this is useful for https://github.com/thin-edge/tedge-standalone

Reply:
Resolved by ab5033e

Reply:
I see that the a central configuration `plugins.path` has been created and it doesn't accept multiple paths.

```sh
$ tedge config list plugin
plugins.path=/usr/share/tedge
diag.plugin_paths=["/usr/share/tedge/diag-plugins"]
```

And the problem with using a value of `/usr/share/tedge` instead of `/usr/share/tedge/log-plugins`, is that can be harder to find the exact folder name where to place the plugin (as the "log-plugins" is hard coded within our code).